### PR TITLE
{lang}[foss/2015a] Tcl 8.6.4 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3775e7eef9e1749c7b4c for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review

File: easybuild/easyconfigs/t/Tcl/Tcl-8.6.4-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ["http://prdownloads.sourceforge.net/tcl"]
+sources = ['%(namelower)s%(version)s-src.tar.gz']
+
+dependencies = [
+    ('zlib', '1.2.8'),
+]
+
+configopts = '--enable-threads EXTRA_INSTALL="install-private-headers"'
```
Comment (wpoely86) [inline easybuild/easyconfigs/t/Tcl/Tcl-8.6.4-foss-2015a.eb]: maybe a comment why the `private` stuff is needed?

**Comment by wpoely86** [comment]
looks good

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
included via a different PR, closing this empty PR

**Pull request closed without merging**